packages/rest/src/rest.server.ts

```diff
@@ -123,6 +123,7 @@ export class RestServer extends Context implements Server, HttpServerLike {
    */
   public requestHandler: HttpRequestListener;
 
+  protected _config: RestServerConfig;
```

Comment:
IMO, this property should not be modified after the constructor has finished, let's mark it as `readonly` please.

Is there any particular reason for keeping the config protected? I mean can we make it `public`?